extensions/amp-story/1.0/amp-story.js

```diff
@@ -790,11 +785,7 @@ export class AmpStory extends AMP.BaseElement {
       if (endsWith(href, '#')) {
         href = href.slice(0, -1);
       }
-      this.win.history.replaceState(
-        (this.win.history && getState(this.win.history)) || {} /** data */,
-        this.win.document.title /** title */,
-        href /** URL */
-      );
+      setHistoryState(this.win, this.win.document.title, href);
```

Comment:
How does this update the URL?

Reply:
Sorry, I thought this was doing something else, I reverted it.